component/attr_cache/attr_cache.go

```diff
@@ -123,10 +123,10 @@ func (ac *AttrCache) Configure() error {
 		return fmt.Errorf("config error in %s [%s]", ac.Name(), err.Error())
 	}
 
-	if conf.Timeout != 0 {
-		ac.cacheTimeout = conf.Timeout
-	} else {
+	if conf.Timeout == nil {
```

Comment:
that IsSet() model if it works better to use that here instead of converting type to pointer